services/common/index.d.ts

```diff
@@ -1,3 +1,5 @@
+import { FieldHint } from "../ui-src/src/components/FieldHint";
```

Comment:
common should have no dependencies on other services...

Reply:
great feedback! ....moved to common with latest commit